toolkit/scss/_instruction-list.scss

```diff
@@ -0,0 +1,105 @@
+@import "_colours.scss";
+@import "_typography.scss";
+@import "_measurements.scss";
+@import "_shims.scss";
+@import "_conditionals.scss";
+
+ul,
+ol {
+  list-style-type: none;
+  padding: 0;
+}
+
+// Bulleted lists
```

Comment:
We also have these styles pulled in from gov.uk elements, e.g.: https://github.com/alphagov/digitalmarketplace-buyer-frontend/blob/master/app/assets/scss/_lists.scss